doc/040_backup.rst

```diff
@@ -146,6 +146,9 @@ args:
 
     $ restic -r /tmp/backup backup --files-from /tmp/files_to_backup /tmp/some_additional_file
 
+Pathes in the listing file should be absolute. Relative pathes will be
```

Comment:
The plural of "path" is "paths", can you please correct that?

I'm not sure if it's a good idea to recommend absolute paths for a file used with `--files-from`. There are plenty of use cases where it's important to specify relative paths. I'm fine with the hint that relative paths will be interpreted in context with the current directory (although that's basic knowledge the reader needs to be familiar with anyway), but I think we should not recommend that the file names in the file should be absolute. Can you remove the first sentence here? Thanks!